Support angular 12

**Describe the bug**

Hello :)
`apollo-angular` has a peer dependency to @angular/core@"^6.1.0 || ^7.0.0 || ^8.0.0 || ^9.0.0 || ^10.0.0 || ^11.0.0". It will be great if it can accept the new angular 12 version

**To Reproduce**
Steps to reproduce the behavior:

Update to angular 12 and try to install apollo-angular (or update it). You will get :

```
npm ERR! Could not resolve dependency:
npm ERR! peer @angular/core@"^6.1.0 || ^7.0.0 || ^8.0.0 || ^9.0.0 || ^10.0.0 || ^11.0.0" from apollo-angular@2.4.0
npm ERR! node_modules/apollo-angular
npm ERR!   apollo-angular@"^2.4.0" from the root project
```

**Environment:**

```
├── @angular/cli@12.0.0
├── @angular/core@12.0.0
├── @apollo/client@3.3.19
├── apollo-angular@2.4.0
├── graphql@15.5.0
└── typescript@4.2.4
```

**Additional context**

Thanks for your work :)

I also have this issue with Angular 11

Could not resolve dependency:
npm ERR! peer @angular/common@"^6.0.0 || ^7.0.0 || ^8.0.0 || ^9.0.0 || ^10.0.0" from apollo-angular-link-http-common@1.10.0
npm ERR! node_modules/apollo-angular-link-http-common
npm ERR!   apollo-angular-link-http-common@"^1.10.0" from the root project

--- a/packages/apollo-angular/CHANGELOG.md
+++ b/packages/apollo-angular/CHANGELOG.md
@@ -9,6 +9,7 @@
 - Typed `gql` and `graphql` tags - both accept two generic types and support `TypedDocumentNode` [`9a8ea5f`](https://github.com/kamilkisiela/apollo-angular/commit/9a8ea5f229cf7937d74332092cb3eba40828b7b1)
 - Add `useMutationLoading` flag [`bc223fe`](https://github.com/kamilkisiela/apollo-angular/commit/bc223fe6487edd35c56ad908e4739580ce69f056)
 - Fix type inference for Mutations [#1659](https://github.com/kamilkisiela/apollo-angular/pull/1659)
+- Declare support for Angular 12
 
 ### v2.4.0
 

--- a/packages/apollo-angular/package.json
+++ b/packages/apollo-angular/package.json
@@ -34,11 +34,11 @@
     "deploy:next": "yarn build && npm publish build --tag next"
   },
   "peerDependencies": {
-    "@angular/core": "^6.1.0 || ^7.0.0 || ^8.0.0 || ^9.0.0 || ^10.0.0 || ^11.0.0",
+    "@angular/core": "^9.0.0 || ^10.0.0 || ^11.0.0 || ^12.0.0",
     "@apollo/client": "^3.0.0",
-    "graphql": ">=0.11.0 <0.14.0 || ^14.0.0 || ^15.0.0",
+    "graphql": "^14.0.0 || ^15.0.0",
     "rxjs": "^6.0.0",
-    "zone.js": ">=0.8.0 <0.12.0"
+    "zone.js": ">=0.10.0 <0.12.0"
   },
   "dependencies": {
     "semver": "^7.0.0",

